src/DependencyInjection/Configuration.php

```diff
@@ -70,6 +86,32 @@ public function getConfigTreeBuilder()
             ->end()
         ;
 
+        $this->addImageSection($rootNode);
+
         return $treeBuilder;
     }
+
+    /**
+     * Configures the `contao.image` section.
+     *
+     * @param ArrayNodeDefinition $rootNode
+     */
+    private function addImageSection(ArrayNodeDefinition $rootNode)
+    {
+        $rootNode
+            ->children()
+                ->arrayNode('image')
+                    ->addDefaultsIfNotSet()
+                    ->children()
+                        ->booleanNode('bypass_cache')
```

Comment:
No hard feelings, but this ain't any better. I suggested `bypass_image_cache`, because the flag is only bypassing the image cache. We also have an internal cache, a search results cache, a page cache, a script cache and the Symfony cache. Unless you are planning to bypass them all if the flag is enabled, the naming is too unspecific.